src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -21315,6 +21314,7 @@ public class ValueTuple<T1, T2, T3> : C<B>
                 Diagnostic(ErrorCode.ERR_NoTypeDef).WithArguments("A", "92872377-08d1-4723-8906-a43b03e56ed3, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null"));
         }
 
+#if NET461
```

Comment:
Why aren't you just ConditionalFact'ing this one?

Reply:
This test uses SigningTestHelpers. That particular class is only available in NET461 ATM hence this must be conditional as well. 

Let me file a bug to follow up on that. 

---
In reply to: [171401523](https://github.com/dotnet/roslyn/pull/25050#discussion_r171401523) [](ancestors = 171401523)